pkg/controller/machine/images.go

```
@@ -0,0 +1,96 @@
+package machine
+
+import (
+	"strings"
+
+	"github.com/samsung-cnct/cma-ssh/pkg/maas"
+)
+
+type image struct {
+	os           string
+	k8sVersion   string
+	instanceType string
+	raw          string
+}
+
+func (i *image) set(key, value string) {
+	switch key {
+	case "os":
+		i.os = value
+	case "k8s":
+		i.k8sVersion = value
+	case "standard":
+		i.instanceType = key
+	case "gpu":
+		i.instanceType = key
+	}
```

It seems as though an errant filename by the standards set here would be silently ignored? Is that correct and if so, is that something that should be addressed? It might be a good idea to at least warn so that the log shows filenames not conforming to these standards exist. It would help debugging potential image problems significantly quicker.

---

https://github.com/samsung-cnct/cma-ssh/pull/108/files#diff-1a9a1e50b1cc40327143fb98d069eb54R31